hphp/runtime/ext/string/ext_string.php

```diff
@@ -1676,7 +1677,7 @@ function levenshtein(string $str1,
  * @return int - Returns the number of matching chars in both strings.
  *
  */
-<<__Native>>
+<<__IsFoldable, __Native>>
 function similar_text(string $first,
                       string $second,
                       mixed &$percent = null): int;
```

Comment:
more reference parameters... I guess this would be ok as long as you check that no parameter is passed for this arg...